client/src/fabric/tensegrity-fabric.ts

```diff
@@ -247,7 +261,42 @@ export class TensegrityFabric {
 
     public iterate(ticks: number): boolean {
         this.disposeOfGeometry()
-        return this.exports.iterate(ticks)
+        const changeHappened = this.exports.iterate(ticks)
+        if (!changeHappened) {
+            return false
+        }
+        const growth = this.growth
+        if (growth) {
+            if (growth.growing.length > 0) {
+                growth.growing = executeGrowthTrees(growth.growing)
+                this.exports.centralize()
+            }
+            if (growth.growing.length === 0) {
```

Comment:
```suggestion
            else {
```

Reply:
or no wait, i'm misunderstanding aren't i. growing.length could be 0 after the code up above

Reply:
No, this is different behavior.  It would miss a case.

Reply:
:+1: